source/codegen/client_helpers.py

```diff
@@ -140,14 +147,21 @@ def _get_param_mechanism(param: dict) -> ParamMechanism:
     return ParamMechanism.COPY
 
 
+def _get_param_default(param: dict) -> str:
+    if param['grpc_type'] == 'nidevice_grpc.SessionInitializationBehavior':
+        return 'nidevice_grpc::SESSION_INITIALIZATION_BEHAVIOR_UNSPECIFIED'
```

Comment:
Note: I wanted the generated _client.h used throughout the tests to have a default for the SessioninitializationBehavior params so I didn't have to specify this default throughout the System tests.